action-check-pods/action.yml

```
@@ -0,0 +1,15 @@
+name: 'Check deployment'
+description: 'GitHub Action for checking if pods and containers are up and running'
+author: 'abuokoene'
+
+inputs:
+  namespace:
+    description: 'The namespace to run action in'
+    required: false
+    default: 'default'
+
+runs:
+  using: 'composite'
+  steps:
+    - run: ${{ github.action_path }}/main.sh
+      shell: bash
```

```suggestion
    - run: ${{ github.action_path }}/main.sh
      shell: bash
      env:
        INPUT_NAMESPACE: ${{ inputs.namespace }}
```

Apparently you need to set env vars as for Composite action they are not set automatically as in Docker/JS actions 🤷‍♂️